akka-stream-tests-tck/src/test/scala/akka/stream/tck/Timeouts.scala

```diff
@@ -10,6 +10,6 @@ object Timeouts {
 
   def publisherShutdownTimeoutMillis: Int = 3000
 
-  def defaultTimeoutMillis: Int = 800
+  def defaultTimeoutMillis: Int = 1600
```

Comment:
I think this will slow down tests too much. The proper fix is awaiting release of reactive-streams/reactive-streams-jvm#309